### DISCUSSION TITLE
How to select the stock location on a quote in the sales module

### DISCUSSION THREAD

**Discussion by Lunatic9297** [OCA/odoo-community.org#164] [Q&A] [answered]
Good morning,
are you doing well ? I have a question about odoo on the quote (Sale).
Before I used another management system I had the possibility to select for each line in the quote (Product-Description-Quantity-Unit Price) to select the stock location for each line, but here I see in '' other information" that it is only possible to do it globally, so I wanted to know if it is possible to do it for each line independently of the other products?

**Comment by Shide** [comment, marked as answer]
You can't do this.
You can select the Warehouse (on the order), route (on the line) if it's previously configured and mix it with categories (using removal strategy)
